module/Database/view/database/email/member-registration.phtml

```diff
@@ -0,0 +1,444 @@
+<?php
+
+declare(strict_types=1);
+
+use Database\Model\ProspectiveMember as ProspectiveMemberModel;
+
+/** @var ProspectiveMemberModel $member */
+
+$restartLink = $this->serverUrl($this->url('member/subscribe/checkout/restart', ['token' => $member->getPaymentLink()->getToken()]));
```

Comment:
And I see this once a session is expired

```bash
stripe post /v1/checkout/sessions/cs_test_a1Uz3BRu4jDJrtqH5HHiNZkHFLGXP5SohDTU1D6hD3NGU8B0Q7Kq0DWkpL/expire
```

![image](https://github.com/GEWIS/gewisdb/assets/17340320/d4494df9-13eb-4385-b1b3-9e6293789ef8)

Reply:
> I don't think this works well currently with the proxy. My test shows localhost:9725, which is the port the nginx proxy is listening on inside the container. The question is what will happen once we pass two proxies

That is because Laminas sucks but it works fine on the actual website:
![image](https://github.com/GEWIS/gewisdb/assets/4983571/0240ec83-5569-47ad-9524-a62e16ad1fbc)

> And I see this once a session is expired

This should also work fine as the webhook is on https://join.gewis.nl and it will thus use that domain.

Reply:
> That is because Laminas sucks but it works fine on the actual website:
> ![image](https://user-images.githubusercontent.com/4983571/258783955-0240ec83-5569-47ad-9524-a62e16ad1fbc.png)

Note that I did not do the `str_replace()` here, only the `serverUrl(url())`.